Docs/source/install/hpc/ookami.rst

```diff
@@ -0,0 +1,126 @@
+.. _building-ookami:
+
+Ookami (Stony Brook)
+====================
+
+The `Ookami cluster <https://www.stonybrook.edu/ookami/>`__ is located at Stony Brook University.
+
+If you are new to this system, please see the following resources:
+
+* `Ookami documentation <https://www.stonybrook.edu/commcms/ookami/support/index_links_and_docs.php>`__
+* Batch system: *TODO*
+* Production directories *TODO*:
+
+  * A high-performance Lustre filesystem provides about 0.8 PByte storage.
+  * ...
+
+We use Ookami as a development cluster for `A64FX <https://www.arm.com/blogs/blueprint/fujitsu-a64fx-arm>`__,
+The cluster also provides a few extra nodes, e.g. two ``Thunder X2`` (ARM) nodes.
+
+
+
+Installation
+------------
+
+Use the following commands to download the WarpX source code and switch to the correct branch:
+
+.. code-block:: bash
+
+   git clone https://github.com/ECP-WarpX/WarpX.git $HOME/src/warpx
+
+We use the following modules and environments on the system (``$HOME/warpx_gcc10.profile``).
+
+.. code-block:: bash
+
+   # please set your project account (not relevant yet)
+   #export proj=<yourProject>
+
+   # required dependencies
+   module load cmake/3.19.0
+   module load gcc/10.3.0
+   module load openmpi/gcc10/4.1.0
+
+   # optional: faster builds (not available yet)
+   #module load ccache
+   #module load ninja
+
+   # optional: for PSATD support (not available yet)
+   #module load fftw
+
+   # optional: for QED lookup table generation support (not available yet)
+   #module load boost
+
+   # optional: for openPMD support
+   #module load adios2  # not available yet
+   #module load hdf5    # only serial
+
+   # compiler environment hints
+   export CC=$(which gcc)
+   export CXX=$(which g++)
+   export FC=$(which gfortran)
+   export CXXFLAGS="-mcpu=a64fx"
+
+
+We recommend to store the above lines in a file, such as ``$HOME/warpx_gcc10.profile``, and load it into your shell after a login:
+
+.. code-block:: bash
+
+   source $HOME/warpx_gcc10.profile
+
+
+Then, ``cd`` into the directory ``$HOME/src/warpx`` and use the following commands to compile:
+
+.. code-block:: bash
+
+   cd $HOME/src/warpx
+   rm -rf build
+
+   cmake -S . -B build -DWarpX_COMPUTE=OMP -DWarpX_OPENPMD=ON -DAMReX_MPI_THREAD_MULTIPLE=FALSE
+   cmake --build build -j 10
+
+   # or (currently better performance)
+   cmake -S . -B build -DWarpX_COMPUTE=NOACC -DWarpX_OPENPMD=ON -DAMReX_MPI_THREAD_MULTIPLE=FALSE
+   cmake --build build -j 10
+
+The general :ref:`cmake compile-time options <building-cmake>` apply as usual.
+
+
+.. _running-cpp-ookami:
+
+Running
+-------
+
+For running on 48 cores of a single node:
+
+.. code-block:: bash
+
+   srun -p short -N 1 -n 48 --pty bash
+   OMP_NUM_THREADS=1 mpiexec -n 48 --map-by ppr:12:numa:pe=1 --report-bindings ./warpx inputs
```

Comment:
It's funny because omitting `--map-by numa` on Ookami is not nearly as bad as omitting it on FX700 !

Anyway, I would provide here also the command to launch WarpX with 4 tasks x 12 threads, which should be:
```
   OMP_NUM_THREADS=12 mpiexec -n 4 --map-by ppr:4:numa:pe=12 --report-bindings ./warpx inputs
```

Reply:
```suggestion
   OMP_NUM_THREADS=1 mpiexec -n 48 --map-by ppr:12:numa:pe=1 --report-bindings ./warpx inputs
   
   # alternatively, using 4 MPI ranks with each 12 threads on a single node:
   OMP_NUM_THREADS=12 mpiexec -n 4 --map-by ppr:4:numa:pe=12 --report-bindings ./warpx inputs
```